### PR TITLE
Update stacks transaction dep in connect package

### PR DESCRIPTION
**Pull request by ahsan-javaid**
## Description
Update the connect dependency to 2.0.0-beta.1 to fix the Serialization of uintCV.
This PR closes the issue #151

## Type of Change
- [ ] New feature
- [X] Bug fix
- [X] API reference/documentation update
- [ ] Other

## Does this introduce a breaking change?
No

## Are documentation updates required?
No

## Testing information
npm install and build the connect to test the fix for Serialization of uintCV.

## Checklist
- [ ] Code is commented where needed
- [ ] Unit test coverage for new or modified code paths
- [ ] `yarn lerna run test` passes
- [ ] Changelog is updated
- [X] Tag 1 of @hstove or @kyranjamie or @aulneau for review

### CODE REVIEW

**Comment by ahsan-javaid** [comment]
@markmhx  This is the fix for #151 as mentioned by @friedger

File: packages/connect/package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "@stacks/auth": "^1.2.3",
     "@stacks/connect-ui": "^5.1.3",
     "@stacks/network": "^1.2.2",
-    "@stacks/transactions": "^1.3.0",
+    "@stacks/transactions": "^2.0.0-beta.1",
```
Comment (friedger) [inline packages/connect/package.json:29]: There is already 2.0.0 and 3.0.0-beta

Comment (ahsan-javaid) [reply packages/connect/package.json:29]: @friedger Should i change it to "2.00" or "3.00-beta". Need your thoughts on this.

Comment (ahsan-javaid) [reply packages/connect/package.json]: Updated to 3.00-beta. Let me know if it makes sense.

Comment (fbwoolf) [reply packages/connect/package.json]: I am not seeing a `3.00-beta` version listed? https://www.npmjs.com/package/@stacks/transactions

I do see a live `2.0.1`, can I make the update and get this tested/merged?

Comment (aulneau) [reply packages/connect/package.json]: Yeah that's the right version

**Comment by fbwoolf** [comment]
I am going to close this PR and make the update in a new PR that isn't forked. Thanks!

**Pull request closed without merging**